src/registry.rs

```diff
@@ -63,13 +65,52 @@ impl RegistryCore {
         // throughout the lifetime of a program.
         Ok(())
     }
+
+    #[allow(unknown_lints)]
```

Comment:
Yes, stable rust is happy. This common is here bacecase clippy warns [`map_entry`](https://github.com/Manishearth/rust-clippy/wiki#map_entry), which I believe is a false alarm.